nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -1483,6 +1626,35 @@ private void checkFieldInitialization(ClassTree tree, VisitorState state) {
       errorBuilder.reportInitErrorOnField(
           uninitSField, state, buildDescription(getTreesInstance(state).getTree(uninitSField)));
     }
+    if (config.autofixIsEnabled()) {
+      fixInitializationErrorsOnControlFlowPaths(state, errorFieldsForInitializer);
+    }
+  }
+
+  private void fixInitializationErrorsOnControlFlowPaths(
```

Comment:
Worth documenting, even as a private method. In particular, worth indicating that this is used as part of autofix logic and expected to run only when `config.autofixIsEnabled()` is `true` (maybe worth a `Preconditions` check?)